src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -107,6 +110,45 @@ public function getRollback($env = null, $target = null)
         return $this->executeRun($command);
     }
 
+    /**
+     * Set the environment
+     *
+     * @param string $environment
+     */
+    public function setEnvironment($environment = null) {
+        if(is_null($environment)) {
```

Comment:
I find it strange that you would define the arguments as optional with `= null`, then throw an exception if the **optional** argument is actually null.